docs/configuration/session.md

```diff
@@ -74,4 +122,9 @@ host: "[fd00:1111:2222:3333::1]"
 
 ## Loading a password from a secret instead of inside the configuration
 
-Password can also be defined using a [secret](../secrets.md).
\ No newline at end of file
+Password can also be defined using a [secret](../secrets.md).
+
+## Redis Sentinel
+
+When using Redis Sentinel, the host specified in the main redis section is added (it will be the first node) to the 
```

Comment:
I'm actually confused by this statement, does the user need to ensure that the first node is what appears in the redis configuration or is this done automatically?
I think the wording needs to be more clear and our specific instructions in this case.

Reply:
It's technically optional. I ensure either the host is specified or the nodes section has a single entry. I'll reword it.